### DISCUSSION TITLE
How to use `prettier/standalone` to format script tag in Vue SFC?

### DISCUSSION THREAD

**Discussion by aiktb** [prettier/prettier#16778] [Q&A] [answered]
```ts
import * as parserBabel from "prettier/parser-babel";
import * as parserHtml from "prettier/parser-html";
import * as parserPostCSS from "prettier/parser-postcss";
import * as prettier from "prettier/standalone";

        prettier
          .format(`<script setup> const a = 1 </script>`, {
            parser: "vue",
            semi: true,
            plugins: [parserBabel, parserHtml, parserPostCSS],
          })
```

`parserPostCSS` works for `style` tag, but `parserBabel` doesn't work for `script` tag.

I expected to see the semicolon added, but it didn't work.

How can I do this?

**Comment by fisker** [comment, marked as answer]
```diff
+ import * as prettierPluginEstree from "prettier/plugins/estree";
import * as parserBabel from "prettier/parser-babel";
import * as parserHtml from "prettier/parser-html";
import * as parserPostCSS from "prettier/parser-postcss";
import * as prettier from "prettier/standalone";

        prettier
          .format(`<script setup> const a = 1 </script>`, {
            parser: "vue",
            semi: true,
-            plugins: [parserBabel, parserHtml, parserPostCSS],
+            plugins: [prettierPluginEstree, parserBabel, parserHtml, parserPostCSS],
          })
```

**Reply by aiktb** [reply]
> ```diff
> + import * as prettierPluginEstree from "prettier/plugin-estree";
> import * as parserBabel from "prettier/parser-babel";
> import * as parserHtml from "prettier/parser-html";
> import * as parserPostCSS from "prettier/parser-postcss";
> import * as prettier from "prettier/standalone";
> 
>         prettier
>           .format(`<script setup> const a = 1 </script>`, {
>             parser: "vue",
>             semi: true,
> -            plugins: [parserBabel, parserHtml, parserPostCSS],
> +            plugins: [prettierPluginEstree, parserBabel, parserHtml, parserPostCSS],
>           })
> ```

Thank you!
It is important to note that the estree plugin is exported from:
```ts
import * as prettierPluginESTree from "prettier/plugins/estree";
```

**Reply by fisker** [reply]
> It is important to note that the estree plugin is exported from:

Thanks updated.
